.github/CODEOWNERS

```
@@ -7,6 +7,9 @@
 # GitHub
 .github/**                    @ezio-melotti @hugovk
 
+# pre-commit
+.pre-commit-config.yaml       @hugovk
```

I've used pre-commit on other projects, and I'm happy to be pinged for reviews :)

```suggestion
.pre-commit-config.yaml       @hugovk @AlexWaygood
```